src/BridgedRoom.ts

```diff
@@ -183,10 +183,10 @@ export class BridgedRoom {
         this.setValue("isPrivate", chan.is_private);
         if (chan.is_channel) {
             this.setValue("slackType", "channel");
-        } else if (chan.is_group) {
-            this.setValue("slackType", "group");
         } else if (chan.is_mpim) {
             this.setValue("slackType", "mpim");
+        } else if (chan.is_group) {
```

Comment:
Can you add a comment here so we remember the ordering is important?

Reply:
I've put the comment in the above if statement instead, since I think it makes more sense there.